assembl/tests/model_tests/test_discussion.py

```diff
@@ -16,3 +16,107 @@ def test_add_discussion(test_session):
     assert d.discussion_locales == ['en', 'fr', 'de']
     assert d.terms_and_conditions.entries[0].value == u"Use the haptic JSON system, then you can quantify the cross-platform capacitor!"  # noqa: E501
     assert d.legal_notice.entries[0].value == u"Use the optical SCSI microchip, then you can generate the cross-platform pixel!"  # noqa: E501
+    test_session.delete(d)
```

Comment:
You need test_session.flush() before the assert, otherwise you don't test the sql part, only the python part.